playbook/workshops/remote_control/README.md

```diff
@@ -0,0 +1,615 @@
+# Remote Control
+
+The objective of this workshop is to learn to wield powerful
+[APIs](http://www.quora.com/What-is-an-API) with minimal code.
+
+This tutorial can be used by complete beginners or more advanced programmers.
+
+**For Beginners**
+
+You will learn how to do really cool things with really simple code. You won't
+understand how it all works and that's totally cool. Your objective is to learn
+how to make things happen. Later you can come back and figure out how it all
+works.
+
+**For More Experienced Folk**
+
+This tutorial will likely show you things that you didn't know that you could
+do. I'd recommend skipping around in the tutorial and find what strikes your
+interest.
+
+## Partners
+
+Before starting this workshop, find a partner! You'll be pair
+programming!
+
+## Quick Demo
+
+We're going to write some code that calls your phone and speaks some
+text.
+
+Ready?
+
+Open [this JS Bin](http://jsbin.com/papawo/6/edit?html,js,console). Leave this
+tab open for the rest of the tutorial.
+
+JS Bin is a website to easily write code for websites.
+
+### Adding your Twilio sid and token
```

Comment:
Also this title in general should conform to [title case](http://www.grammar-monster.com/lessons/capital_letters_title_case.htm)